erpnext_com/www/docs/user/manual/en/setting-up/articles/prepared-report.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
the red box used to highlight buttons needs be placed better. currently it's not uniform around the buttons